gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
Just make a `keys/` folder in your project folder and add it on `.gitignore` .

Or you can make use of env variables and for contributors can ask to generate a key for them if they wants to contribute.